packages/scandipwa/src/component/ResetAttributes/ResetAttributes.container.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import PropTypes from 'prop-types';
+import { PureComponent } from 'react';
+import { connect } from 'react-redux';
+
+import { formatPrice } from 'Util/Price';
+
+import ResetAttributes from './ResetAttributes.component';
+
+/** @namespace Component/ResetAttributes/Container/mapStateToProps */
+export const mapStateToProps = (state) => ({
+    currency_code: state.ConfigReducer.currencyData.current_currency_code
+});
+
+/** @namespace Component/ResetAttributes/Container/mapDispatchToProps */
+// eslint-disable-next-line no-unused-vars
+export const mapDispatchToProps = (dispatch) => ({});
+
+/** @namespace Component/ResetAttributes/Container */
+export class ResetAttributesContainer extends PureComponent {
+    static propTypes = {
+        availableFilters: PropTypes.objectOf(PropTypes.shape({
+            attribute_label: PropTypes.any,
+            attribute_type: PropTypes.string.isRequired,
+            attribute_code: PropTypes.any,
+            attribute_values: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+            attribute_options: PropTypes.objectOf(PropTypes.shape({
+                label: PropTypes.string.isRequired,
+                value_string: PropTypes.string.isRequired
+            }))
+        }).isRequired).isRequired,
+        customFiltersValues: PropTypes.any.isRequired,
+        currency_code: PropTypes.string.isRequired
+    };
+
+    containerProps = () => ({
+        filtersData: this.filterResetItems()
+    });
+
+    containerFunctions = () => ({
+        resetFilters: this.resetFilters
+    });
+
+    getPriceFilterLabel(filterValue) {
```

Comment:
Please move this logic to price util, do same for CategoryConfigurableAttributes